app/src/main/java/net/gaast/giggity/Schedule.java

```diff
@@ -948,6 +969,11 @@ public void startElement(String uri, String localName, String qName,
 			if (localName.equals("vevent")) {
 				eventData = new HashMap<>();
 			}
+			if (localName.equals("attendee")) {
+				if (atts.getValue("role") == "REQ-PARTICIPANT")
+					eventData.put("speaker", new String(atts.getValue("cn")));
+					// TODO: handle more than one?
```

Comment:
Yes, that is for example how multiple speakers are provided in FOSDEM'S iCal file.